sdk/jobs/pipelines/1c_pipeline_with_hyperparameter_sweep/pipeline_with_hyperparameter_sweep.ipynb

```diff
@@ -79,31 +80,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# 2. Create command component\n",
```

Comment:
As this is a example right after 1b_pipeline_with_python_function_components, I simplify the component definition section